setup.cfg

```diff
@@ -1,72 +0,0 @@
-[metadata]
-name = psycopmlutils
-description = Code for working with the PSYCOP cohort
-long_description = file: README.md
-long_description_content_type = text/markdown
-classifiers =
-    Environment :: Console
-    Intended Audience :: Developers
-    Intended Audience :: Science/Research
-    Topic :: Scientific/Engineering
-    Topic :: Scientific/Engineering :: Artificial Intelligence
-    Operating System :: POSIX :: Linux
-    Operating System :: MacOS :: MacOS X
-    Operating System :: Microsoft :: Windows
-    Programming Language :: Python :: 3.7
-    Programming Language :: Python :: 3.8
-    Programming Language :: Python :: 3.9
```

Comment:
important metadata is not included.

Reply:
"Important" might be an overstatement for internal tooling 😉 

I've added it in the next commit 👍